src/test/testHelpers/config.spec.js

```diff
@@ -0,0 +1,15 @@
+import chai from 'chai';
```

Comment:
can we rename this from `config.spec.js` to `config.js` because it is not a test file and no describe and it block define

Reply:
yes, done... Thank you